lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php

```diff
@@ -77,8 +78,8 @@ protected function initialize(InputInterface $input, OutputInterface $output) :
         $helperSet = $this->getHelperSet() ?: new HelperSet();
 
         if ($helperSet->has('configuration') && $helperSet->get('configuration') instanceof ConfigurationHelper) {
-            /** @var MigrationsConfigurationHelper $configHelper */
             $configHelper = $helperSet->get('configuration');
+            assert($configHelper instanceof MigrationsConfigurationHelper);
```

Comment:
Is this consider a BC?

Reply:
I wouldn't consider this as BC break... Although I think this shouldn't be necessary since there's an `instanceof` validation on the `if`

Reply:
since `master` has not been tagged, BC breaks are not an issue here.

Reply:
Amazing! If the PR is okay, feel free to merge it, otherwise I'll merge it next week between the holidays :)

Reply:
Please do not merge this yet, it will create not necessary conflicts with the remaining PR I have on this repo